_includes/drawer.html

```diff
@@ -1,11 +1,14 @@
 <div class="mdl-layout__drawer">
-  <span class="mdl-layout-title">Apereo Foundation</span>
+  <span class="mdl-layout-title">
+    <a href="{{ site.baseurl }}{% link index.md %}">Apereo Foundation</a
+  ></span>
```

Comment:
Is closing `>` intended to be on next line?

Reply:
Good catch. No, that wasn't intended, it was an off-by-one-character placed newline.

Clicking precisely is hard. I should really learn to stay on the keyboard more.